pkgs/applications/science/misc/root/default.nix

```diff
@@ -57,36 +60,37 @@
 
 let
 
-  _llvm_9 = llvm_9.overrideAttrs (prev: {
-    patches = (prev.patches or [ ]) ++ [
-      (fetchpatch {
-        url = "https://github.com/root-project/root/commit/a9c961cf4613ff1f0ea50f188e4a4b0eb749b17d.diff";
-        stripLen = 3;
-        hash = "sha256-LH2RipJICEDWOr7JzX5s0QiUhEwXNMFEJihYKy9qWpo=";
-      })
-    ];
+  clang_patched = llvmPackages_13.clang.cc.overrideAttrs (prev: {
+    version = "ROOT-patches-13";
+    sourceRoot = null;
+    src = fetchgit {
+      url = "http://root.cern/git/clang.git";
```

Comment:
This repository will not be maintained going forward. We have since switched to a fork of the LLVM monorepo, but it there is no tagged version in there that corresponds to 6.28/04 (this will change for future releases). For the time being, I think it would be better to continue using `builtin_clang`.

Reply:
Thanks for the tip. Fixed.